community/kernel/src/docs/dev/batchinsert.asciidoc

```diff
@@ -57,3 +76,12 @@ component=neo4j-kernel
 source=examples/BatchInsertDocTest.java
 tag=configFileInsert
 ----
+
+=== Importing into an existing database ===
+
+Although it's a less common use case, the batch inserter can also be used to import data into an existing database.
```

Comment:
It's does sound like you still can have it running, so I think it needs to be more explicit.